client/src/pages/Home.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import AppBar from '@material-ui/core/AppBar';
+import Tab from '@material-ui/core/Tab';
+import Tabs from '@material-ui/core/Tabs';
+import Box from '@material-ui/core/Box';
+import { makeStyles } from '@material-ui/core/styles';
+
+// TODO: Insert event,appointment components, make responsive
+
+const useStyles = makeStyles((theme) => ({
+
+    dashboard: {
+        padding: "70px 150px 0px 150px",
+    },
+
+    title: {
+        marginBottom: "12px"
+    },
+
+    appbar: {
+        backgroundColor: theme.bgcolor
+    },
+
+    tabIndicator: {
+        height: 5,
+    },
+
+    tab: {
+        marginRight:"25px",
+        paddingLeft:"0px",
+        color:theme.palette.secondary.main,
+    }
+
+}));
+
+function Home() {
+  const classes = useStyles();
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+
+  return (
+    <div>
+        <Grid container className={classes.dashboard}>
+            <Grid item xs={12} className={classes.title} >
+                <Typography variant="h5">My CalendApp</Typography>
+            </Grid>
+
+            <Grid item xs={12}>
+            <AppBar position="static" className={classes.appbar} elevation={0}>
+                <Tabs value={value} onChange={handleChange} textColor="primary" indicatorColor="primary"
+                      classes={{ indicator: classes.tabIndicator }} >
+                  <Tab className={classes.tab} label={<Typography variant="subtitle1">EVENT TYPES</Typography>} />
+                  <Tab className={classes.tab} label={<Typography variant="subtitle1">SCHEDULED EVENTS</Typography>} />
+                </Tabs>
+            </AppBar>
+            </Grid>
+        </Grid>
+
+        {value===0 && <div>Event page</div>}
+        {value===1 && <div>Appointment page</div>}
```

Comment:
Works for now, but it's not sustainable to use arbitrary numbers to determine which component to render. It makes for confusing code really fast. Creating routes with `react-router-dom` might be a more sustainable choice.

Reply:
Actually yes Tony if it is possible to use routing here that would be great!
If you try and it's not simple to implement, you can just go back to this approach, but try using the `TabPanel` approach that is in the MaterialUI documentation under the Tabs component. We can discuss this more in standup